config/locales/en.yml

```diff
@@ -1357,6 +1357,7 @@ en:
           enable: "Allow data sharing"
           disable: "Stop sharing"
           loading: "Loading"
+          need_to_be_manager: "Only managers can connect apps."
```

Comment:
Would this be a more helpful explanation? Helps work out how to solve the problem.
```suggestion
          need_to_be_manager: "You cannot connect an app because you are not a manager of this enterprise."
```

Reply:
That would probably be more helpful. I kept it short because it fits better in that space. It's only seen by super-admins and therefore I wouldn't put too much effort into the message.